protocol/statetracker/consumer_state_tracker.go

```diff
@@ -93,6 +93,19 @@ func (cst *ConsumerStateTracker) RegisterForPairingUpdates(ctx context.Context,
 	}
 }
 
+func (cst *ConsumerStateTracker) RegisterForPairingStakeEntriesUpdates(ctx context.Context, pairingUpdatable updaters.PairingStakeEntriesUpdatable, specId string) {
```

Comment:
no need for that after the pairing updater comment.

Reply:
Removed in 437c61458270cbd7bbf5f4246e0f76407bec3139